tests/test_core.py

```diff
@@ -135,27 +126,39 @@ def rename_coords(ds):
 
     col = intake.open_esm_datastore(esmcol_path)
     col_sub = col.search(**query)
-
     dsets = col_sub.to_dataset_dict(zarr_kwargs={'consolidated': True}, preprocess=rename_coords)
     _, ds = dsets.popitem()
     assert 'latitude' in ds.dims
     assert 'longitude' in ds.dims
 
 
+def test_to_dataset_dict_w_cmip6preprocessing(pangeo_cmip6_col):
+    pytest.importorskip('cmip6_preprocessing')
+    from cmip6_preprocessing.preprocessing import combined_preprocessing
+
+    cat = pangeo_cmip6_col.search(
```

Comment:
Ah very nice. I wanted to add some tests with the actual cloud data to cmip6_preprocessing too. Ill orient myself on this example.